packages/web-react/src/components/SplitButton/UncontrolledSplitButton.tsx

```diff
@@ -0,0 +1,54 @@
+import React, { useState } from 'react';
+import { UncontrolledSplitButtonProps } from '../../types';
+import { Button } from '../Button';
+import { Dropdown, DropdownPopover, DropdownTrigger } from '../Dropdown';
+import { Icon } from '../Icon';
+import { VisuallyHidden } from '../VisuallyHidden';
+import SplitButton from './SplitButton';
+
+const defaultProps: Partial<UncontrolledSplitButtonProps> = {
+  dropdownPlacement: 'bottom-end',
+  dropdownIconName: 'chevron-down',
+};
+
+const UncontrolledSplitButton = (props: UncontrolledSplitButtonProps) => {
+  const propsWithDefaults = { ...defaultProps, ...props };
+  const {
+    buttonIconName,
+    buttonLabel,
+    buttonOnClick,
+    children,
+    dropdownIconName,
+    dropdownLabel,
+    dropdownPlacement,
+    id,
+    isDisabled,
+    isDropdownLabelHidden,
+    isLabelHidden,
+    ...restProps
+  } = propsWithDefaults;
+  const [openDropdownState, setOpenDropdownState] = useState(false);
+
+  return (
+    <SplitButton id={id} {...restProps} isDisabled={isDisabled}>
+      <Button onClick={buttonOnClick}>
+        {buttonIconName && <Icon name={buttonIconName} marginRight={!isLabelHidden ? 'space-400' : undefined} />}
```

Comment:
```suggestion
        {buttonIconName && <Icon name={buttonIconName} marginRight={isLabelHidden ?  undefined  : 'space-400'} />}
```

Or maybe `marginRight={isLabelHidden && 'space-400'}`